sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -100,7 +100,7 @@ namespace Azure { namespace Storage { namespace Blobs {
   private:
     std::string GetResumeToken() const override
     {
-      // Not supported
+      // Not sud
```

Comment:
Not sure what caused this change.
```suggestion
      // Not supported
```